internal/cli/deployment_list.go

```diff
@@ -71,6 +71,7 @@ func (c *DeploymentListCommand) Run(args []string) int {
 	client := c.project.Client()
 
 	err := c.DoApp(c.Ctx, func(ctx context.Context, app *clientpkg.App) error {
+		app.UI.Output("%s", app.Ref().Application, terminal.WithHeaderStyle())
```

Comment:
Same as other about the output not being valid json in this case.

Reply:
I'm also thinking for the json case, for all projects, instead of returning two separate json structures separated by newlines, we should probably collect and wrap them into a single array for each item listed? Maybe that's a future enhancement 🤔

Reply:
Hm, we could do that by storing the json on the command, returning nothing in the `DoApp` if flag.json, and then returning the collected json outside of that loop?